notebooks/01-variables.ipynb

```diff
@@ -17,12 +17,12 @@
   },
```

Comment:
<p>Este esta genial! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoLatinas/Intro-to-python/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>